packages/installer/src/certificates.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2021 Opstrace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  V1CertificaterequestResourceType,
+  V1CertificateResourceType
+} from "@opstrace/kubernetes";
+import { log } from "@opstrace/utils";
+
+//
+// Note: workaround for https://github.com/opstrace/opstrace/issue/151 until
+// https://github.com/jetstack/cert-manager/issues/3594 is resolved.
+//
+// If the https-cert certificate is in the failed state then delete it. The
+// controller will recreate the resource thereby triggering a restart of the
+// certificate request process.
```

Comment:
Yeeeeess, thank you for this code comment. Now it took me 10 seconds to understand what the idea here is. Love that.